modules/page_base.py

```diff
@@ -278,6 +284,7 @@ def get_element(
         if not multiple:
             found_element = self.driver.find_element(*selector)
             if "doNotCache" not in element_data["groups"]:
+                logging.info("Caching...")
```

Comment:
Would appreciate more specifics in the caching message, for example: "Caching sign-in-bar-button..." (same goes for not caching!)

Reply:
done